skypy/halo/mass.py

```diff
@@ -1,38 +1,319 @@
-r'''Halo mass module.
-
-This module provides methods to sample the  masses of dark matter halos.
+r'''Halo mass sampler.
+This code samples halos from their mass function.
 
 Models
 ======
-
 .. autosummary::
    :nosignatures:
    :toctree: ../api/
 
    press_schechter
-
+   halo_mass_function
+   halo_mass_sampler
+   sheth_tormen_collapse_function
+   press_schechter_collapse_function
+   press_schechter_mass_sampler
 '''
 
 import numpy as np
-
+from scipy import integrate
+from functools import partial
+from astropy import units as u
+from skypy.power_spectrum import growth_function
 from skypy.utils.random import schechter
 
+__all__ = [
+    'halo_mass_function',
+    'halo_mass_sampler',
+    'sheth_tormen_collapse_function',
+    'press_schechter_collapse_function',
+    'press_schechter_mass_function',
+    'press_schechter',
```

Comment:
```suggestion
    'press_schechter',
    'sheth_tormen',
```
Also add a partial halo_mass_sampler for sheth tormen using the params values quoted in their paper